assets/js/base/components/cart-checkout/shipping-rates-control/index.tsx

```diff
@@ -65,7 +65,9 @@ const Packages = ( {
 					packageData={ packageData }
 					collapsible={ collapsible }
 					collapse={ collapse }
-					showItems={ showItems }
+					showItems={
+						showItems || packageData?.shipping_rates?.length > 1
```

Comment:
This logic will evaluate to true when you have multiple shipping rates within a single package, not overall.

Reply:
Logic should be `showItems || packages.length > 1`